pycbc/waveform/bank.py

```diff
@@ -633,12 +632,20 @@ def get_decompressed_waveform(self, tempout, index, f_lower=None,
 
         from pycbc.waveform.waveform import props
         from pycbc.waveform import get_waveform_filter_length_in_time
+       
+        # Get the template hash corresponding to the template index taken in as argument
+        tmplt_hash = self.table.template_hash[index]
+
+        # Read the compressed waveform from the bank file
+        compressed_waveform = pycbc.waveform.compress.CompressedWaveform.from_hdf(
+                                self.filehandler, tmplt_hash,
+                                load_now=self.load_compressed_now)
```

Comment:
Just set `load_now=True`; again, you have to load the sample points now since they are being used immediately, so it doesn't make sense to delay it.